src/SFA.DAS.EmployerAccounts.UnitTests/Queries/GetAccountEmployerAgreementsRemove/WhenIGetAccountEmployerAgreementsToRemove.cs

```diff
@@ -18,47 +21,65 @@ namespace SFA.DAS.EmployerAccounts.UnitTests.Queries.GetAccountEmployerAgreement
 {
     public class WhenIGetAccountEmployerAgreementsToRemove : QueryBaseTest<GetAccountEmployerAgreementsRemoveQueryHandler,GetAccountEmployerAgreementsRemoveRequest, GetAccountEmployerAgreementsRemoveResponse>
     {
+        private Mock<IDasRecruitService> _dasRecruitService;
         private Mock<IEmployerAgreementRepository> _repository;
         private Mock<IHashingService> _hashingService;
         private Mock<IEmployerCommitmentApi> _commitmentsApi;
         public override GetAccountEmployerAgreementsRemoveRequest Query { get; set; }
         public override GetAccountEmployerAgreementsRemoveQueryHandler RequestHandler { get; set; }
         public override Mock<IValidator<GetAccountEmployerAgreementsRemoveRequest>> RequestValidator { get; set; }
-
+        public Vacancy Vacancy { get; set; }
```

Comment:
Does this need to be public?

Reply:
Changed to Private. Does not need to be private.